spec/components/previews/toggler_dropdown_component_preview/examples.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="row">
+  <div class="col">
+
+    <h2>title: 'Title'</h2>
+
+    <%= render TogglerDropdownComponent.new(title: 'Title') do %>
+      Content in here
+    <% end %>
+
+    <h2>surround: false</h2>
+    <p>Just renders content as if component not there</p>
+
+    <%= render TogglerDropdownComponent.new(title: 'Title', surround: false) do %>
```

Comment:
I get an error as there's no `surround` keyword defined in the constructor

Reply:
After I fixed this to see the examples, I noticed that if you toggle the second one then the first one opens. Think the target attributes need to adjusting so that we could have multiple on a page (although not sure how much we'll be using this in user facing pages).

Reply:
Thanks - I changed the name of the parameter just before the final commit and forgot about the preview...

Have fixed the second toggle issue though thanks!